src/AppBundle/Resources/data/posts/2016-02-10.yml

```diff
@@ -0,0 +1,41 @@
+name: "first-1-3-release-candidate"
+parent: "/news"
+title: "First Symfony CMF 1.3 Release Candidate Available"
+label: false
+format: markdown
+template: cms/news_detail.hml.twig
+publish_start_date: "2016-02-10"
+body: |
+    The release of Symfony CMF 1.3 is coming closer with the availability of
+    release candidates for all packages. Stable releases can be expected in the
+    coming weeks.
+
+    After the 1.2 release, the activity of the Symfony CMF project has dropped,
+    resulting in no new release in 2014. With this new release, the CMF is back
+    in running with full Symfony 2.8 and PHP 7 support.
+
+    ## Important new features
+    
+    The biggest features in this release are:
+
+     * Symfony 3 support for the Routing and Testing component;
```

Comment:
expectation management: this is easy to misread as symfony 3 support for everything. we should be more explicit that not all components got symfony 3 support yet.

i would put the php 7 support as first line, because that is generally available